ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -1056,18 +1056,8 @@ func (r *RayClusterReconciler) buildHeadPod(ctx context.Context, instance rayv1.
 	return pod
 }
 
-// TODO (kevin85421): It is better to use `CRDType` as the return type.
-func getCreator(instance rayv1.RayCluster) string {
-	if instance.Labels == nil {
-		return ""
-	}
-	creatorName, exist := instance.Labels[utils.RayOriginatedFromCRDLabelKey]
-
-	if !exist {
-		return ""
-	}
-
-	return creatorName
+func getCreatorCRDType(instance rayv1.RayCluster) utils.CRDType {
+	return utils.GetCRDType(instance.Labels[utils.RayOriginatedFromCRDLabelKey])
```

Comment:
If the label is `nil`, `instance.Labels[utils.RayOriginatedFromCRDLabelKey]` will return an empty string.